server/api_design/validate_contracts.py

```
@@ -0,0 +1,80 @@
+def main():
+    files = glob.glob(os.path.join(CONTRACTS_DIR, "*.json"))
+    for f in files:
+        if os.path.basename(f) == "errors.json":
+            continue
+        validate_contract(f)
+
+
+if __name__ == "__main__":
+    try:
+        main()
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)